### PR TITLE
Add CERN as adopter

### PR DESCRIPTION
**Pull request by OrlinVasilev**
Add CERN as an adopter

depends on https://github.com/goharbor/website/pull/482

### CODE REVIEW

**Comment by OrlinVasilev** [comment]
@rochaporto for visibility :) thank you!!

**Review by wy65701436 — APPROVED** [review]
lgtm

File: ADOPTERS.md [hunk 1]
```diff
@@ -36,7 +36,7 @@ be added to this list as they transition to production deployments.
 <a href="http://www.yanrongyun.com" target="_blank" border="0"><img alt="Yanrongyun" src="https://raw.githubusercontent.com/goharbor/website/main/docs/img/adopters/Yanrong.jpg" height="40"></a>&nbsp; &nbsp; &nbsp; &nbsp;
 <a href="https://anchore.com" target="_blank" border="0"><img alt="Anchore" src="https://raw.githubusercontent.com/goharbor/website/main/docs/img/adopters/anchore_logo.png" height="40"></a>&nbsp; &nbsp; &nbsp; &nbsp;
 <a href="https://www.dynatrace.com/" target="_blank" border="0"><img alt="Dynatrace" src="https://raw.githubusercontent.com/goharbor/website/main/static/img/logos/users-partners/dynatrace-logo.png"></a>&nbsp; &nbsp; &nbsp; &nbsp;
-
```
Comment (rochaporto) [inline ADOPTERS.md:39]: It's best if we can skip the logo (its use is a bit complicated, we would need to make an explicit request that can take long).

Comment (OrlinVasilev) [reply ADOPTERS.md:39]: Oh didn't know that, will remove from the PR and we can add later when we have approval!

**Comment by rochaporto** [comment]
lgtm

**Pull request merged**